Либерия

LordNest opened the pull request:
Данный Пулл Реквест добавляет FTV Liberia, торговое судно Вольного Торгового Союза

Closes #49 

<details>
<summary>Чейнджлог</summary>

```yml
🆑
rscadd: Добавлена авей-карта "Либерия"
/🆑
```

</details>

- [x] Pull Request полностью завершен, мне не нужна помощь чтобы его закончить.
- [x] Я внимательно прочитал все свои изменения и багов в них не нашел.
- [x] Я запускал сервер со своими изменениями локально и все протестировал.

LordNest committed:
Liberia initial commit

Adds Liberia and related content

LordNest committed:
Account Bugfix

Fixed absence of away accounts in ATMs

LordNest committed:
Mining

Adds random mining maps into rotation

SuhEugene commented:
Какой ревью... Оно тесты даже не проходит...

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

LordNest committed:
Unit tests fix

SuhEugene committed:
Заполнение карты Сьерра - Часть 2 (#82)

Добавляет отсутствующий контент. Что именно - в ченджлоге

Closes #80
Closes #89

LordNest committed:
tables_fix

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

LordNest committed:
Conflict_resolve

LordNest committed:
Merge branch 'Away_branch' of https://github.com/LordNest/SierraBay12 into Away_branch

LordNest committed:
Mapfixes

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

SuhEugene committed:
Комментарий обозначения замены

SuhEugene committed:
Юджин дурак

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

LordNest committed:
This dosen't work

Commiting only to make this shit visible

LordNest committed:
Merge branch 'SierraBay:dev220' into Away_branch

LordNest committed:
Here goes the cash

Fixed absence of money from initial economy skill for merchant

LordNest committed:
Merge branch 'Away_branch' of https://github.com/LordNest/SierraBay12 into Away_branch

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+// Submap datum and archetype.
+/datum/job/submap/merchant
```
`/datum/job/submap/merchant` -> `/datum/job/submap/merchant_leader`
`/datum/job/submap/merchant_trainee` -> `/datum/job/submap/merchant`
Чтобы жобка сабмапы соответствовала аутфиту
Некрасиво же

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	announced = FALSE
+	min_skill = list(
+		SKILL_FINANCE = SKILL_ADEPT,
+		SKILL_PILOT	  = SKILL_BASIC
```
А давай либо сверху тоже таб будет, либо здесь вместо таба - три пробела...

```suggestion
		SKILL_PILOT   = SKILL_BASIC
```

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	allowed_ranks = list(
+		/datum/mil_rank/civ/civ
+	)
+	latejoin_at_spawnpoints = 1
```
Були - `TRUE`/`FALSE`

```suggestion
	latejoin_at_spawnpoints = TRUE
```

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	selection_color = "#515151"
+	ideal_character_age = 30
+	minimal_player_age = 7
+	create_record = 0
```
Були - `TRUE`/`FALSE`

```suggestion
	create_record = FALSE
```

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	selection_color = "#515151"
+	ideal_character_age = 20
+	minimal_player_age = 0
+	create_record = 0
```
Були

```suggestion
	create_record = FALSE
```

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	allowed_ranks = list(
+		/datum/mil_rank/civ/civ
+	)
+	latejoin_at_spawnpoints = 1
```
```suggestion
	latejoin_at_spawnpoints = TRUE
```

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	for(var/obj/item/briefcase_item in sec_briefcase)
+		qdel(briefcase_item)
+	for(var/i=money_to_put, i>0, i--)
+		new /obj/item/spacecash/bundle/c2500 (sec_briefcase)
```
```suggestion
		new /obj/item/spacecash/bundle/c2500(sec_briefcase)
```

Просто чтоб по фен-шую

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	var/obj/item/storage/secure/briefcase/sec_briefcase = new(H)
+	for(var/obj/item/briefcase_item in sec_briefcase)
+		qdel(briefcase_item)
+	for(var/i=money_to_put, i>0, i--)
```
```suggestion
	for(var/i = money_to_put, i > 0, i--)
```
Фен-Шуй требует чтобы кровать стояла головой на север *вроде бы*.

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	name = OUTFIT_JOB_NAME("Merchant Assistant")
+	uniform = /obj/item/clothing/under/suit_jacket/tan
+	shoes = /obj/item/clothing/shoes/brown
+//	pda_type = /obj/item/modular_computer/pda
```
```suggestion
```

TODO нема, значит комментарий не нужен

SuhEugene commented on mods/maps/liberia/liberia_jobs.dm:
```diff
@@ -0,0 +1,194 @@
+	)
+	give_psionic_implant_on_join = FALSE
+	skill_points = 24
+	economic_power = 10 // We use splitted from station account system, which need lover economic_power to not break things
```
Это проценты?
Если да, то это выглядит как кринжа какая-то...

LordNest replied:
Это не проценты, это статичное значение, не процентное.

SuhEugene reviewed (changes requested):
![UwUThinking](https://cdn.discordapp.com/emojis/896477408339849227.png?size=32)

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

LordNest committed:
Update mods/maps/liberia/liberia_jobs.dm

Co-authored-by: SuhEugene <32931701+SuhEugene@users.noreply.github.com>

SuhEugene committed:
Переименование путей

`/datum/job/submap/merchant` -> `/datum/job/submap/merchant_leader`
`/datum/job/submap/merchant_trainee` -> `/datum/job/submap/merchant`

SuhEugene committed:
Ancient evil

SuhEugene reviewed (approved):
Галка
